pallets/fragments/src/lib.rs

```diff
@@ -66,32 +119,66 @@ pub struct FragmentClass<TFungibleAsset, TAccountId, TBlockNum> {
 	pub created_at: TBlockNum,
 }
 
+/// **Struct** of a **Fragment Instance**
+///
+/// Notes from Giovanni:
+/// 
+/// #### Remarks
+///  
+/// * On purpose not storing owner because:
+///   * Big, 32 bytes
+///   * Most of use cases will definitely already have the owner available when using this structure, as likely going thru `Inventory` etc.
 #[derive(Encode, Decode, Clone, scale_info::TypeInfo, Debug, PartialEq)]
-pub struct InstanceData<TBlockNum> {
+pub struct FragmentInstance<TBlockNum> {
 	/// Next owner permissions, owners can change those if they want to more restrictive ones, never more permissive
+	/// **Actions** (encapsulated in a `FragmentPerms` bitflag enum) **allowed to be done** to the **Fragment Instance**
+	/// (e.g edit, transfer etc.)
 	pub permissions: FragmentPerms,
```

Comment:
So, basically there are permissions on both `FragmentInstance `and `FragmentDefinition` (which contained in the instance).
This is because if the `FragmentInstance `is transferred to another account/owner, the permissions of the instance can be changed by that new account/owner (without changing the `FragmentDefinition` permissions)?

Reply:
Likewise we should prob discuss this

Reply:
Yes @alebaffa , you are correct.

I have further improved the documentation for how permissions work in the commit titled "clarified how permissions work in Pallet Fragments". Let me know if it's written more clearly now. Thanks

Reply:
I have further improved the documentation for how permissions work in the commit titled "clarified how permissions work in Pallet Fragments". Let me know if it's written more clearly now. Thanks